packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -453,6 +472,14 @@ export class GarbageCollector implements IGarbageCollector {
             && !gcOptions.disableGC
         );
 
+        const currentVersion = pkgVersion;
```

Comment:
Use `pkgVersion` directly instead of creating a variable since its only used once.